platforms/core-execution/execution-e2e-tests/src/integTest/groovy/org/gradle/integtests/NestedInputKotlinImplementationTrackingIntegrationTest.groovy

```diff
@@ -191,7 +191,7 @@ class NestedInputKotlinImplementationTrackingIntegrationTest extends AbstractInt
         """
 
         when:
-        if (['1.4', '1.5', '1.6'].contains(kotlinLanguageVersion)) {
+        if (['1.6', '1.7'].contains(kotlinLanguageVersion)) {
```

Comment:
❓ why it was changed for 1.7?

Reply:
Because Kotlin 2.1 changes the supported language versions
1.4 is not supported anymore
1.7 is now deprecated